app/src/main/java/com/example/kakaowebtoon/presentation/ui/home/HomeScreen.kt

```diff
@@ -35,7 +36,8 @@ fun HomeScreen(
     ) {
         Text(
             text = "HomeScreen 입니다",
-            color = KakaoWebtoonTheme.colors.white
+            color = KakaoWebtoonTheme.colors.white,
+            modifier = modifier.noRippleClickable { onNavigateToSearch() }
```

Comment:
여기도 위 코멘트와 동일하게 이렇게 되어야하지 않을까요?
```suggestion
            color = KakaoWebtoonTheme.colors.white,
            modifier = Modifier.noRippleClickable { onNavigateToSearch() }
```